cmd/ttn-lw-cli/commands/contact_info.go

```diff
@@ -109,6 +109,8 @@ var contactInfoFlags = util.FieldFlags(&ttnpb.ContactInfo{})
 var (
 	errContactInfoExists           = errors.DefineAlreadyExists("contact_info_exists", "contact info already exists")
 	errMatchingContactInfoNotFound = errors.DefineAlreadyExists("contact_info_not_found", "matching contact info not found")
+	errNoReference                 = errors.DefineInvalidArgument("missing_ref", "missing reference")
+	errNoToken                     = errors.DefineInvalidArgument("missing_token", "missing validation token")
```

Comment:
"no" vs "missing"

Reply:
Also beware that these errors are defined in `cmd/ttn-lw-cli/commands`, so they should be somewhat specific. Token and reference can mean different things in different contexts, so:

```go
errNoValidationToken = errors.DefineInvalidArgument("no_validation_token", "no validation token")
```